build.gradle

```diff
@@ -6,7 +6,7 @@ buildscript {
 }
 
 plugins {
-    id 'com.android.application' version '8.0.2' apply false
-    id 'com.android.library' version '8.0.2' apply false
+    id 'com.android.application' version '7.3.0' apply false
+    id 'com.android.library' version '7.3.0' apply false
```

Comment:
This was causing issues for jitpack. I think we can fix this later with a jitpack config file